### PR TITLE
[debian] Create a virtual package for gramine-dcap

### PR DESCRIPTION
**Pull request by oshogbo**
## Description of the changes 

In this commit, we define a virtual package for gramine-dcap. The goal of it is to smoothly transit users from the `gramine-dcap` to Gramine package.

## Useful links:
* https://wiki.debian.org/RenamingPackages
* https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-and-replacing-packages-replaces
* https://rpmdeb.com/devops-articles/how-to-create-local-debian-repository/

## How to test this PR? 
Tested upgrade from 1.3.1 to 1.4.

- - -
This change is [Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)

### CODE REVIEW

**Review by dimakuv — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (3 more required), not enough approvals from different teams (2 more required, approved so far: ) (waiting on @oshogbo)

___
*[`-- commits` line 6 at r1](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkDnkP3pAU7AgY4QLG):*
I would put everything in quotes, so that it's clear these are names of packages.

```
[debian] Create a virtual package for `gramine-dcap`

Define a virtual package for `gramine-dcap`; the goal is to smoothly
transit users from `gramine-dcap` package to `gramine` package.
```

**Review by oshogbo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (3 more required), not enough approvals from different teams (2 more required, approved so far: ) (waiting on @dimakuv)

___
*[`-- commits` line 6 at r1](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkDnkP3pAU7AgY4QLG):*
<details><summary><i>Previously, dimakuv (Dmitrii Kuvaiskii) wrote…</i></summary><blockquote>

I would put everything in quotes, so that it's clear these are names of packages.

```
[debian] Create a virtual package for `gramine-dcap`

Define a virtual package for `gramine-dcap`; the goal is to smoothly
transit users from `gramine-dcap` package to `gramine` package.
```
</blockquote></details>

Done.

**Review by mkow — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 2 unresolved discussions, not enough approvals from maintainers (3 more required), not enough approvals from different teams (2 more required, approved so far: ), "squash! " found in commit messages' one-liners (waiting on @dimakuv and @oshogbo)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkq5N48ayTD18V9g3E:-NNkq5N48ayTD18V9g3F:b-xnskn5) (no related file):*
Blocking until @woju approves this PR.

___

**Review by dimakuv — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "squash! " found in commit messages' one-liners (waiting on @oshogbo)

**Review by woju — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "squash! " found in commit messages' one-liners (waiting on @jinengandhi)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkq5N48ayTD18V9g3E:-NNqCcvlEt9XbEmx3SiZ:bmn8iay) (no related file):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

Blocking until @woju approves this PR.
</blockquote></details>

LGTM, but I won't approve until @jinengandhi tests this update flow with `1.4~rc3`, which will carry this commit.

___

**Review by woju — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "squash! " found in commit messages' one-liners (waiting on @jinengandhi)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkq5N48ayTD18V9g3E:-NNr0IJy1bi7Da21j8sI:byklbsa) (no related file):*
<details><summary><i>Previously, woju (Wojtek Porczyk) wrote…</i></summary><blockquote>

LGTM, but I won't approve until @jinengandhi tests this update flow with `1.4~rc3`, which will carry this commit.
</blockquote></details>

I've carried this change in #1151, so depending on how you prefer to have this merged, you can either leave it on #1151, or eventually merge this and I'll drop the commit from #1151, which is already bigger than I'd like to have a release PR.

___

**Review by dimakuv — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "squash! " found in commit messages' one-liners (waiting on @jinengandhi)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkq5N48ayTD18V9g3E:-NNuhLiX7se_LxnmL69B:bxuz6iv) (no related file):*
<details><summary><i>Previously, woju (Wojtek Porczyk) wrote…</i></summary><blockquote>

I've carried this change in #1151, so depending on how you prefer to have this merged, you can either leave it on #1151, or eventually merge this and I'll drop the commit from #1151, which is already bigger than I'd like to have a release PR.
</blockquote></details>

I'd prefer to have everything in one single PR, like @woju did with #1151. So I vote for closing this PR and proceeding with Woju's PR.

___

**Comment by oshogbo** [comment]
Fine with me.

**Review by mkow — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "squash! " found in commit messages' one-liners (waiting on @jinengandhi)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkq5N48ayTD18V9g3E:-NNwKCws06VFpBdByosm:b-8i8w0v) (no related file):*
> I'd prefer to have everything in one single PR

Wait, why? That PR already has 5 commits and this change is standalone. Moving it there only makes things more complicated.

___

**Review by mkow — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/gramineproject/gramine/1162)* status: all files reviewed, 1 unresolved discussion, not enough approvals from maintainers (2 more required), not enough approvals from different teams (1 more required, approved so far: Intel), "squash! " found in commit messages' one-liners (waiting on @jinengandhi)

*[a discussion](https://reviewable.io/reviews/gramineproject/gramine/1162#-NNkq5N48ayTD18V9g3E:-NO4oLmZ6Wi4h4ZT-20t:bs1kq0q) (no related file):*
<details><summary><i>Previously, mkow (Michał Kowalczyk) wrote…</i></summary><blockquote>

> I'd prefer to have everything in one single PR

Wait, why? That PR already has 5 commits and this change is standalone. Moving it there only makes things more complicated.
</blockquote></details>

For the record / in case anyone reads this in the future: @woju didn't just move this commit to #1151, he also modified it significantly. I.e. don't look at this PR review, it's not relevant.

___

**Pull request closed without merging**
